tests/python/contrib/test_hexagon/test_software_pipeline_async.py

```diff
@@ -46,18 +47,33 @@ def plus_one_ref(a):
     return plus_one_primfunc, plus_one_ref
 
 
-@tvm.testing.requires_hexagon
-def test_software_pipeline_with_cache_read(hexagon_launcher, compute, outer, inner, dtype, scope):
+@tvm.testing.fixture
+def schedule(compute, sched, scope):
     sch = tir.Schedule(compute[0])
-    root = sch.get_block("root")
+
     compute_block = sch.get_block("compute")
     cache_read_block = sch.cache_read(compute_block, 0, scope)
 
     i, _ = sch.get_loops(compute_block)
     sch.compute_at(cache_read_block, i)
-    sch.annotate(i, "software_pipeline_stage", [0, 1])
-    sch.annotate(i, "software_pipeline_order", [0, 1])
-    sch.annotate(i, "software_pipeline_async_stages", [0])
+
+    if sched == "cache_read":
```

Comment:
Does it make sense for the sake of completeness to test a cache_write pipeline where `sch.annotate(i, "software_pipeline_async_stages", [1])`?